src/python/pants/backend/javascript/package/package-lock.json

```diff
@@ -0,0 +1,4164 @@
+{
+  "name": "pants",
```

Comment:
the integration test uses `ham` for the name which does not match this. Does that matter?

Reply:
I'd expect it to, but npm seemingly do not validate/error out when this does not match during a clean-install.